client/app/components/catalogs/catalog-editor.component.js

```diff
@@ -0,0 +1,265 @@
+/* eslint camelcase: "off" */
+(function() {
+  'use strict';
+
+  angular.module('app.components')
+    .component('catalogEditor',  {
+      templateUrl: 'app/components/catalogs/catalog-editor.html',
+      bindings: {
+        catalog: "=",
```

Comment:
No action: must these be two way bound,  or can catalog and serviceTemplates be one way bound (`<`)take advantage of component outputs ('&')?